src/components/interactive/DropdownButton.jsx

```diff
@@ -3,10 +3,12 @@ import PropTypes from 'prop-types';
 import cx from 'classnames';
 import CaretDownIcon from 'lib/icons/react/CaretDownIcon';
 
-const DropdownButton = ({ children, ...rest }) => (
+const DropdownButton = ({ children, isFullWidth = false, ...rest }) => (
```

Comment:
I'm assuming this is a more stylistic thing that you do on `cbi-site` where you call out `defaultProps` values just for  clarity. In general I'd personally just utilize the `isFullWidth` being falsy... don't think it should stop the PR but it may be worth aligning philosophies there at some point.

Reply:
I've done it this way since they are [planning on deprecating `defaultProps` for functional components](https://github.com/reactjs/rfcs/pull/107) in the future. I've been doing it in cbi-site and it's just become a habit now.